device/mellanox/x86_64-mlnx_msn2410-r0/plugins/psuutil.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python
+
+#############################################################################
+# Mellanox
+#
+# Module contains an implementation of SONiC PSU Base API and
+# provides the PSUs status which are available in the platform
+#
+#############################################################################
+
+import os.path
+
+try:
+    from sonic_psu.psu_base import PsuBase
+except ImportError, e:
+    raise ImportError (str(e) + "- required module not found")
+
+class PSUutil(PsuBase):
```

Comment:
Suggest changing case of class name to `PsuUtil` to match CamelCase of `SfpUtil`

See matching suggestion here: https://github.com/Azure/sonic-utilities/pull/152#discussion_r152106460

Reply:
Done